doc/playground.lua

```diff
@@ -0,0 +1,150 @@
+                },
+            },
+        },
+    },
+}
+vshard.storage.cfg(cfg, box.info().uuid)
+vshard.router.cfg(cfg)
+vshard.router.bootstrap()
+
+-- Create the 'customers' space.
+box.once('customers', function()
+    box.schema.create_space('customers', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+            {name = 'age', type = 'number'},
+        }
+    })
+    box.space.customers:create_index('primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        },
+    })
+    box.space.customers:create_index('bucket_id', {
+        parts = {
+            {field = 2, type = 'unsigned'},
+        },
+        unique = false,
+    })
+    box.space.customers:create_index('age', {
+        parts = {
+            {field = 4, type = 'number'},
+        },
+        unique = false,
+    })
+
+    -- Fill the space.
+    box.space.customers:insert({1, 477, 'Elizabeth', 12})
+    box.space.customers:insert({2, 401, 'Mary', 46})
+    box.space.customers:insert({3, 2804, 'David', 33})
+    box.space.customers:insert({4, 1161, 'William', 81})
+    box.space.customers:insert({5, 1172, 'Jack', 35})
+    box.space.customers:insert({6, 1064, 'William', 25})
+    box.space.customers:insert({7, 693, 'Elizabeth', 18})
+end)
+
+-- Create the developers space.
+box.once('developers', function()
+    box.schema.create_space('developers', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+            {name = 'surname', type = 'string'},
+            {name = 'age', type = 'number'},
+        }
+    })
+    box.space.developers:create_index('primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        },
+    })
+    box.space.developers:create_index('bucket_id', {
+        parts = {
+            {field = 2, type = 'unsigned'},
+        },
+        unique = false,
+    })
+    box.space.developers:create_index('age_index', {
+        parts = {
+            {field = 5, type = 'number'},
+        },
+        unique = false,
+    })
+    box.space.developers:create_index('full_name', {
+        parts = {
+            {field = 3, type = 'string'},
+            {field = 4, type = 'string'},
+        },
+        unique = false,
+    })
+
+    -- Fill the space.
+    box.space.developers:insert({1, 477, 'Alexey', 'Adams', 20})
+    box.space.developers:insert({2, 401, 'Sergey', 'Allred', 21})
+    box.space.developers:insert({3, 2804, 'Pavel', 'Adams', 27})
+    box.space.developers:insert({4, 1161, 'Mikhail', 'Liston', 51})
+    box.space.developers:insert({5, 1172, 'Dmitry', 'Jacobi', 16})
+    box.space.developers:insert({6, 1064, 'Alexey', 'Sidorov', 31})
+end)
+
+-- Initialize crud.
+crud.init_storage()
+crud.init_router()
+
+-- Start a console.
+console.start()
```

Comment:
![image](https://user-images.githubusercontent.com/20455996/170455403-56a9ba6a-8922-4b5d-a7ee-c112a3297dc4.png)

Having explicit `null` as a second return val is interesting. Is this a side effect of `console`?

Reply:
It is from `sharding.wrap_select_method` (the same for insert). I spin on it for some time, but then decided that it is not a bug. The second return value is an error.

TBH, I would change it to make visually better in a console (not in this PR), but I'll not file an issue, because it is a thing of taste.